common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -0,0 +1,220 @@
+#### Local Types ####
```

Comment:
Works better as the time is money art IMHO.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105626175-a80c4f00-5e47-11eb-8d35-474ed2d1f1b4.png)

</details>